tests/modelbench_tests/test_consistency_checker.py

```diff
@@ -63,22 +101,40 @@ def test_normal_run(tmp_path, basic_benchmark_run):
         assert subchecker.warnings == []
 
 
+def test_entities_collected(tmp_path, basic_benchmark_run):
+    checker = init_checker_for_journal(tmp_path, basic_benchmark_run)
+
+    assert sorted(checker.suts) == sorted(["sut1", "sut2"])
```

Comment:
The RHS value is already sorted. Does it need to be explicitly sorted?

Reply:
lol true